controllers/grid/pubIds/PubIdExportIssuesListGridHandler.inc.php

```diff
@@ -0,0 +1,228 @@
+		AppLocale::requireComponents(
+			LOCALE_COMPONENT_APP_COMMON,
+			LOCALE_COMPONENT_APP_SUBMISSION,
+			LOCALE_COMPONENT_PKP_SUBMISSION,
+			LOCALE_COMPONENT_APP_EDITOR
+		);
+
+		// Fetch the authorized roles and determine if the user is a manager.
+		$authorizedRoles = $this->getAuthorizedContextObject(ASSOC_TYPE_USER_ROLES);
+		$this->_isManager = in_array(ROLE_ID_MANAGER, $authorizedRoles);
+
+		$pluginCategory = $request->getUserVar('category');
+		$pluginPathName = $request->getUserVar('plugin');
+		$this->_plugin = PluginRegistry::loadPlugin($pluginCategory, $pluginPathName);
+
+		// Grid columns.
+		$cellProvider = new PubIdExportIssuesListGridCellProvider($this->_plugin, $authorizedRoles);
+		$this->addColumn(
+			new GridColumn(
+				'identification',
+				'issue.issue',
+				null,
+				null,
+				$cellProvider,
+				array('html' => true,
+						'alignment' => COLUMN_ALIGNMENT_LEFT)
+			)
+		);
+		$this->addColumn(
+			new GridColumn(
+				'published',
+				'editor.issues.published',
+				null,
+				null,
+				$cellProvider,
+				array('html' => true,
+						'alignment' => COLUMN_ALIGNMENT_LEFT)
+			)
+		);
+		$this->addColumn(
+			new GridColumn(
+				'pubId',
+				null,
+				$this->_plugin->getPubIdDisplayType(),
+				null,
+				$cellProvider,
+				array('alignment' => COLUMN_ALIGNMENT_LEFT,
+						'width' => 15)
+				)
+		);
+		$this->addColumn(
+			new GridColumn(
+				'status',
+				'common.status',
+				null,
+				null,
+				$cellProvider,
+				array('alignment' => COLUMN_ALIGNMENT_LEFT,
+						'width' => 10)
+			)
+		);
+	}
+
+
+	//
+	// Implemented methods from GridHandler.
+	//
+	/**
+	 * @copydoc GridHandler::getRowInstance()
+	 */
+	protected function getRowInstance() {
+		return new GridRow();
+	}
+
+	/**
+	 * @copydoc GridHandler::initFeatures()
+	 */
+	function initFeatures($request, $args) {
+		import('lib.pkp.classes.controllers.grid.feature.selectableItems.SelectableItemsFeature');
+		import('lib.pkp.classes.controllers.grid.feature.PagingFeature');
+		return array(new SelectableItemsFeature(), new PagingFeature());
+	}
+
+	/**
+	 * @copydoc GridHandler::getRequestArgs()
+	 */
+	function getRequestArgs() {
+		return array_merge(parent::getRequestArgs(), array('category' => $this->_plugin->getCategory(), 'plugin' => basename($this->_plugin->getPluginPath())));
+	}
+
+	/**
+	 * @copydoc GridHandler::isDataElementSelected()
+	 */
+	function isDataElementSelected($gridDataElement) {
+		return false; // Nothing is selected by default
+	}
+
+	/**
+	 * @copydoc GridHandler::getSelectName()
+	 */
+	function getSelectName() {
+		return 'selectedIssues';
+	}
+
+	/**
+	 * @copydoc GridHandler::getFilterForm()
+	 */
+	protected function getFilterForm() {
+		return 'controllers/grid/pubIds/pubIdExportIssuesGridFilter.tpl';
+	}
+
+	/**
+	 * @copydoc GridHandler::renderFilter()
+	 */
+	function renderFilter($request, $filterData = array()) {
+		$statusNames = $this->_plugin->getStatusNames();
+		$filterData = array(
```

Comment:
Just in case subclasses try to use the `$filterData` parameter, best to `array_merge` the data you want here with the stuff that's passed into the function.